contracts/SpokePool.sol

```diff
@@ -768,17 +970,16 @@ abstract contract SpokePool is
      * @param repaymentChainId Chain of SpokePool where relayer wants to be refunded after the challenge window has
      * passed. Will receive inputAmount of the equivalent token to inputToken on the repayment chain.
      */
-    function fillV3Relay(V3RelayData calldata relayData, uint256 repaymentChainId)
-        public
-        override
-        nonReentrant
-        unpausedFills
-    {
+    function fillV3Relay(
+        V3RelayData calldata relayData,
+        uint256 repaymentChainId,
+        bytes32 repaymentAddress
```

Comment:
Should we add a `fillV3Relay` overloaded function where the default repayment address is `msg.sender`? Could this avoid some breaking changes for existing fillers to give them time to migrate to new functions? Or will they already be broken by all the changes in the `V3RelayData` struct, for example the address -> bytes32 changes.

Reply:
Yes, the changes in `V3RelayData` would also be breaking changes, so an update would still be necessary. I’m leaning towards only adding backward compatibility to functions that are called by other contracts, like `depositV3`.

Reply:
I think the changes in V3Relay might not be as breaking necessarily since address can be implictly converted to bytes32. But I take your point.

@pxrl wdyt about this?